pkg/metrics/metrics.go

```diff
@@ -23,32 +23,59 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1alpha1"
+	"sigs.k8s.io/kueue/pkg/constants"
 )
 
 type AdmissionResult string
+type ClusterQueueStatus int
```

Comment:
why not just define it as string from the get got?

Reply:
much simpler :)

I had copied the constants from the cache package